lib/Model/bundle.js

```diff
@@ -59,7 +59,7 @@ function serializeCollections(root) {
       var doc = collection.docs[id];
       var shareDoc = doc.shareDoc;
       var snapshot;
-      if (shareDoc) {
+      if (shareDoc && shareDoc.version !== null) {
```

Comment:
We can only ignore the doc if *both* version and type are null. It is a valid state that there might be a type (a created document) but we might not have a version yet if we haven't gotten one from the server.

```
if (shareDoc) {
  if (shareDoc.type == null && shareDoc.version == null) {
    snapshot = undefined;
  } else {
    ...
  }
}
```

Reply:
thanks i'll make this update